scripts/build.sh

```diff
@@ -1,120 +0,0 @@
-#!/bin/bash
-
-# env vars passed into client & server apps (done by webpack)
-export APP_NAME="A db-agnostic, React-based web application starter boilerplate"
-
-LAST_COMMIT=$(git log -1 --pretty=format:"%h")
-GIT_DIRTY_STATE=$(git diff --quiet || echo "*")
-export BUILD_SIGNATURE=$LAST_COMMIT$GIT_DIRTY_STATE
-
-# some lines are commented out as they are optional features
-
-START_TIME=$(date +%s)
-
-mkdir -p ./build/{server,public/css}
-
-THIS_DIR=`dirname $0`
-BUILD_DIR=`realpath "$THIS_DIR/../build"`
-
-export WEBPACK_CLIENT_APP_OUTPUT_DIR="$BUILD_DIR/public"
-export WEBPACK_CLIENT_APP_OUTPUT_FILENAME="js/main.js"
-
-export WEBPACK_SERVER_APP_OUTPUT_DIR="$BUILD_DIR/server"
-export WEBPACK_SERVER_APP_OUTPUT_FILENAME="main.js"
-
-export SERVER_APP="$WEBPACK_SERVER_APP_OUTPUT_DIR/$WEBPACK_SERVER_APP_OUTPUT_FILENAME"
-export CLIENT_APP="$WEBPACK_CLIENT_APP_OUTPUT_DIR/$WEBPACK_CLIENT_APP_OUTPUT_FILENAME"
-
-# client-side
-build_client() {
-
-  printf "\nbuilding for client-side...\n\n"
-
-  # webpack will be under `watch` mode if WATCH=true, hence the parallel process
-  { webpack --config ./src/client/webpack.config.js >&1 ; } &
-
-  # uncomment if you want to use Atlaskit
-  # cp -r ./node_modules/@atlaskit/css-reset/dist/bundle.css ./build/public/css/css-reset.css
-
-  # uncomment if you want to use Blueprint
-  # cp -r ./node_modules/@blueprintjs/datetime/lib/css/* ./build/public/css/
-  # cp -r ./node_modules/normalize.css/normalize.css ./build/public/css/
-  # cp -r ./node_modules/@blueprintjs/core/lib/css/* ./build/public/css/
-  # cp -r ./node_modules/@blueprintjs/icons/lib/css/* ./build/public/css/
```

Comment:
- [ ] add these back to the new build script (`_shared-env.sh`) - we don't want to lose these instructions